docs/requirements.txt

```diff
@@ -1,4 +1,5 @@
 sphinx<2
 pyscaffold==2.5.10
+mistune==0.8.4
```

Comment:
Where and how is this dependency used?

Reply:
It is used by m2r (the line below) and m2r does not set version and the project is set to read-only. This meant that m2r wanted to install a version of mistune that was not backwards compatible thus breaking m2r for us.

Reply:
I want to remove m2r when we are updating to support newer python versions